src/Engine/Graphics/Renderer/OpenGLRenderer.cpp

```diff
@@ -4926,10 +4854,7 @@ bool OpenGLRenderer::Reinitialize(bool firstInit) {
 
     if (firstInit) {
         // initiate shaders
-        if (!InitShaders()) {
-            logger->warning("shader initialisation has failed!");
-            return false;
-        }
+        InitShaders();
```

Comment:
If ReloadShaders returns a bool we could maintain this exit? Otherwise shader failure message box can be closed and game will continue as normal until it tries to use the invalid shader when it will assert.